src/SFA.DAS.AdminService.Web/Views/Application/Sequence.cshtml

```diff
@@ -71,6 +95,17 @@
                             <dd class="govuk-summary-list__value">@Model.FinancialDueDate.ToSfaShortDateString()</dd>
                         </div>
                     }
+                    @if (Model.IsWithdrawal)
+                    {
+                        <div class="govuk-summary-list__row">
+                            <dt class="govuk-summary-list__key">Contact details</dt>
+                            <dd class="govuk-summary-list__value">
+                                @Model.ContactName
+                                <br />
+                                <a href="mailto:@Model.ContactEmail" class="govuk-link">@Model.ContactEmail</a>
```

Comment:
There may be a thing about no using mailto - I've had this in the past query with deen or design team?

Reply:
Not sure - this is how it is in the design

Reply:
Removed mailto: